container/container.go

```diff
@@ -201,11 +201,11 @@ func (a *AvroContainerWriter) constructor() string {
 }
 
 func (a *AvroContainerWriter) constructorDef() string {
-	return fmt.Sprintf(containerWriterConstructorTemplate, a.constructor(), a.name(), a.schemaVariable(), a.name())
+	return fmt.Sprintf(containerWriterConstructorTemplate, a.constructor(), a.schemaVariable(), a.name())
 }
 
 func (a *AvroContainerWriter) writeRecordDef() string {
-	return fmt.Sprintf(containerWriterWriteTemplate, a.name(), a.record.GoType())
```

Comment:
No, the method name is still WriteRecord. Or did I not understand your question?